README.md

```diff
@@ -298,7 +298,7 @@ descriptors:
     descriptors:
       - key: subkey      
         rate_limit:
-          -  requests_per_unit: 300
+             requests_per_unit: 300
```

Comment:
I will look into the CI issues but can you change this to 2 space indent which I think is what is in the rest of the file? Also I think the same issue happens above? Thank you.